client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
+				) &&
```

Comment:
You can use `isNewUser` from [Guided Tours contexts](https://github.com/Automattic/wp-calypso/blob/c256a69bd15a139dc06cb446f28b53aab9a60125/client/state/ui/guided-tours/contexts.js#L20) :)